src/handshake/certificate_verify.rs

```diff
@@ -7,8 +10,12 @@ use super::CryptoBuffer;
 #[derive(Debug)]
 #[cfg_attr(feature = "defmt", derive(defmt::Format))]
 pub struct CertificateVerifyRef<'a> {
+    #[cfg(feature = "webpki")]
     pub(crate) signature_scheme: SignatureScheme,
+    #[cfg(feature = "webpki")]
     pub(crate) signature: &'a [u8],
+    #[cfg(not(feature = "webpki"))]
+    _todo: PhantomData<&'a ()>,
 }
```

Comment:
Guarding this behind 'webpki' doesn't look right. The intention is that you provide the TlsVerifier trait with a custom implementation. Instead these fields should be public (not sure why they're not).

Reply:
Done, they are now public fields instead.